cidc_api/models.py

```diff
@@ -20,10 +22,70 @@
 from sqlalchemy.orm.session import Session
 from sqlalchemy.dialects.postgresql import JSONB, ARRAY, BYTEA
 from sqlalchemy.ext.declarative import declarative_base
+from eve_sqlalchemy.config import DomainConfig, ResourceConfig
 
 from cidc_schemas import prism
 
-BaseModel = declarative_base()
+## Constants
+ORGS = ["CIDC", "DFCI", "ICAHN", "STANFORD", "ANDERSON"]
+ROLES = [
+    "cidc-admin",
+    "cidc-biofx-user",
+    "cimac-biofx-user",
+    "cimac-user",
+    "developer",
+    "devops",
+    "nci-biobank-user",
+]
+
+# See: https://github.com/CIMAC-CIDC/cidc-schemas/blob/master/cidc_schemas/schemas/artifacts/artifact_core.json
+ARTIFACT_CATEGORIES = [
+    "Assay Artifact from CIMAC",
+    "Pipeline Artifact",
+    "Manifest File",
+    "Other",
+]
+ASSAY_CATEGORIES = [
+    "Whole Exome Sequencing (WES)",
```

Comment:
For the sake of consistency it would be nice to have `ASSAY_CATEGORIES`, `ARTIFACT_CATEGORIES` to be parsed from the schema directly and not repeated here.

E.g. in `dm_update` schema branch I already removed "Other" from both of those.

Reply:
Yeah, consistency will be a problem here. I worry, though, that populating these constants dynamically from `cidc_schemas` may lead to inconsistencies between the database schema and the python data models. Since the database migrations need to be generated by hand, a dynamic change to `ASSAY_CATEGORIES` won't automatically be reflected in the Postgres schema.

Alternatively, we could just make `DownloadableFiles.assay_category` and  `DownloadableFiles.artifact_category` string-type rather than enum type, relying on `cidc_schemas` to do the validation for us.